### PR TITLE
Merge release 1 7

### PR DESCRIPTION
**Pull request by albertshau**

### CODE REVIEW

**Comment by albertshau** [comment]
only conflicts were pom versions

File: .gitmodules [hunk 1]
```diff
@@ -18,28 +18,8 @@
 [submodule "wrangler-transform"]
 	path = wrangler-transform
 	url = https://github.com/hydrator/wrangler-transform.git
-	branch = develop
+	branch = release/2.0
```
Comment (CuriousVini) [inline .gitmodules]: This should be develop.

**Comment by sagarkapare** [comment]
LGTM

**Review by CuriousVini — CHANGES_REQUESTED** [review]

**Comment by CuriousVini** [comment]
lgtm

**Pull request merged**